packages/react-aria-components/src/DropZone.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, PressEvents} from '@react-types/shared';
+import {ButtonContext} from './Button';
+import {ContextValue, Provider, RenderProps, SlotProps, useContextProps, useRenderProps} from './utils';
+import {DropOptions, mergeProps, useButton, useClipboard, useDrop, useFocusRing, useHover, useId, usePress, VisuallyHidden} from 'react-aria';
+import {filterDOMProps, useHasChild, useIsMobileDevice} from '@react-aria/utils';
+import {InputContext} from './Input';
+import {isVirtualDragging} from '@react-aria/dnd';
```

Comment:
see daniel's [comment](https://github.com/adobe/react-spectrum/pull/4451#discussion_r1202851237)

i left it as is but would like other opinions on this

Reply:
I think I'll revise my opinion here actually. I don't think `isVirtualDragging` should be exported from the `react-aria` monopackage actually, this is probably fine